### PR TITLE
[WIP] Slight Changeling Rework

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Ling is a bad gamemode. This aims to make it somewhat less bad by reworking the abilities.

## What Changed?
- Adjusted required players from 2 to 10. Adjusted required changelings from 1 to 4. Changeling should be a team based gamemode and should not run if there is measly amount of players. This should also help with the lowpop problem Aurorastation has.
- Reworked Absorption mechanics by adding Absorb DNA - Lesser and Absorb DNA - Greater. Absorb DNA - Greater is like the regular SUCC mechanics which gives you 5 genetic points and refills your chemicals, Absorb DNA - Lesser is similar to Extract DNA String however it costs no chemicals and grants 1 genetic point. This should make changelings deviate from murderbone mechanics.
- Crewmembers on painkillers won't be able to feel stealth stings.
- Reworked some powers to work with lesser/monkey form.
- Adjusted the genome unlock cost for some of the more deadlier powers.
- Removed Death String. It's balance does not fit in an HRP setting and I don't think I've seen it used.

## WIP
All this is subject to change and requires feedback. None of this has been tested yet so it may not work.

## Feedback Thread
https://forums.aurorastation.org/viewtopic.php?f=18&t=11834

### CODE REVIEW

**Comment by BurgerLUA** [comment]
!review

**Comment by BurgerLUA** [comment]
Forgot this PR existed. Haven't tested it yet, but I would like for it to get reviewed.

File: code/game/gamemodes/changeling/changeling_powers.dm [hunk 1]
```diff
@@ -136,35 +139,71 @@ var/global/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","E
 
 	return
 
-//Absorbs the victim's DNA making them uncloneable. Requires a strong grip on the victim.
-//Doesn't cost anything as it's the most basic ability.
-/mob/proc/changeling_absorb_dna()
+/mob/proc/changeling_can_steal_dna(var/mob/living/carbon/human/T)
+
+	if(!istype(T))
+		src << "<span class='warning'>[T] is not compatible with our biology.</span>"
```
Comment (BurgerLUA) [inline code/game/gamemodes/changeling/changeling_powers.dm]: to_chat, i know

**Comment by skull132** [comment]
@BurgerLUA check the WIP and review required flags. Close if stale.

**Pull request closed without merging**